src/steamship/agents/examples/logs/shiplog--2023-07-15--00:02:11.log

```diff
@@ -0,0 +1,7 @@
+[Client] Creating/Fetching workspace with handle/id: default/None.
```

Comment:
did you mean to include these logs?  should we add a .gitignore ?

Reply:
Lol you were so fast with your slack notifications you caught the PR before I removed the logs :)

Added a .gitignore!